response_operations_ui/exceptions/error_codes.py

```diff
@@ -14,6 +14,9 @@ class ErrorCode(Enum):
     API_OIDC_CREDENTIALS_ERROR = "E0006"
     API_TIMEOUT_ERROR = "E0007"
 
+    # HTTP Error Codes
+    NOT_FOUND = "404"
```

Comment:
Fine for the moment, but intended to be an non-HTTP code specific to Rops. We can change this again